docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+# Introduction to ERC721 (NFT) Smart Contracts
+
+This tutorial will start you with a basic ERC721 (NFT) smart contract on the Avalanche Network, regardless of your previous development experience. We will deploy our NFT on the Avalanche Fuji Testnet and view it on the Snowtrace Testnet Explorer. Note that these are not transferable to the Mainnet. However, once you feel comfortable launching your project, you can do so on Avalanche Mainnet and list it on an NFT marketplace.
+
+The following tools will be used during this tutorial:
+
+- [Pinata](https://www.pinata.cloud/): To store your NFT images and metadata.
+- [OpenZeppelin’s Wizard](https://wizard.openzeppelin.com/): to create the ERC721 smart contract.
+- [Remix IDE](https://remix-project.org/): To edit the code and deploy it to Fuji.
+- [Avalanche Testnet Faucet](https://faucet.avax.network/): To fund the deployment.
+- [MetaMask browser Extension](https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn): To process transactions related to funding and deploying the smart contract.
```

Comment:
Can Core be used in this tutorial? If yes, we should add both Core and Metamask.

Reply:
Will address as a separate PR